project/Sheng08/README.md

```diff
@@ -0,0 +1,253 @@
+- **Continuous Integration**
+  - [Github Actions](https://github.com/features/actions)
+
+## Schedule
+
+| Week                                                       | Schedule                                                                                                                                                                                                                                                                            |
+| ---------------------------------------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| **Week 1 (11/6)**<br>Initial Setup and Environment Configuration | 1. Define project scope for OrientMesh Toolkit.<br>2. Set up GitHub repository and version control with Git.<br>3. Install the required environment (C++, Python, `pybind11`, `.obj` file parser library, etc.).                                                                    |
+| **Week 2 (11/13)**<br>Mesh Data Handling                         | 1. Design and implement `MeshDataHandler` class in C++.<br>2. Test the class with various `.obj` files to ensure proper parsing and representation.<br>3. Start documentation on the class functions using Markdown.                                                                |
+| **Week 3 (11/20)**<br>OBB Extraction                             | 1. Research and decide on the optimal algorithm for OBB extraction.<br>2. Begin implementation of the `OBBExtractor` class in C++.<br>3. Test the OBB extraction on sample 3D models and refine the algorithm.<br>4. Document the methods and results.                              |
+| **Week 4 (11/27)**<br>Python Binding & Interface                 | 1. Explore and set up `pybind11` for creating Python bindings for C++ code.<br>2. Implement the `OOBInterface` class and Python bindings.<br>3. Test the integration of C++ and Python code and ensure seamless function call transitions.                                          |
+| **Week 5 (12/4)**<br>Interactive GUI Development                 | 1. Design the GUI layout and functionalities.<br>2. Start the implementation of GUI using `PyQt` in Python.<br>3. Integrate the GUI with the core library and ensure real-time visualization of OBB and meshes.                                                                     |
+| **Week 6 (12/11)**<br>Testing & Debugging                        | 1. Create test cases using Google Test for C++ and pytest for Python.<br />2. Identify bugs or inefficiencies in the code and refine them.<br />3. Optimize the performance of the extraction algorithm and GUI interactions.<br />4. Update documentation with the recent changes. |
```

Comment:
Creating tests late delays your development.  You'll experience how it happens.

Reply:
After giving it much thought, I've adjusted the development schedule. Now, I plan to write tests immediately after completing the development each week. Thank you for your suggestion.

Reply:
Do not resolve discussions.  It's harder for me to keep track of it.